lib/ApplicationFeatures/ApplicationServer.cpp

```diff
@@ -188,9 +186,6 @@ void ApplicationServer::run(int argc, char* argv[]) {
   // off even in the prepare phase
   disableDependentFeatures();
 
-  // permanently drop the privileges
-  dropPrivilegesPermanently();
```

Comment:
I understand that this PR removes the calls to the useless `dropPrivilegesTemporarily()` method. This is fine, because `dropPrivilegesTemporarily()` and `raisePrivilegesTemporarily()` did not thing.
But `dropPrivilegesPermanently` seems to do something, so I am not sure if removing it is the right thing. Can we preserve it?